uvicorn/middleware/debug.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, app, scope):
         try:
             asgi = self.app(self.scope)
             await asgi(receive, self.send)
-        except:
+        except BaseException as exc:
```

Comment:
I think there’s some adjustements needed in this block - eg `raise` May need to be `raise exc from None`. I’d double check how the behaviour looks in a browser with a simple app that raises an exception: possible our test cases are dealing with ensuring the correct behaviour fully. Similarly I *expect* `traceback.format_exc()` will now need to be passed the exception instance explicitly or something similar.